api/caching/tasks.py

```diff
@@ -111,7 +111,7 @@ def update_storage_usage_cache(target_id):
     ).files.aggregate(sum=models.Sum('versions__size'))['sum'] or 0
 
     key = cache_settings.STORAGE_USAGE_KEY.format(target_id=target_id)
-    cache.set(key, storage_usage_total, cache_settings.NEVER_TIMEOUT)
+    cache.set(key, storage_usage_total, cache_settings.FIVE_MIN_TIMEOUT)
```

Comment:
@Johnetordoff, I'm not sure why, but the cached storage value has an infinite expiration date when run in a celery task, despite it being set to five minutes here.  If I do the cache updating synchronously, not relying on a task, it sets correctly.

Inside celery task:
![screen shot 2019-01-31 at 2 31 36 pm](https://user-images.githubusercontent.com/9755598/52083517-415c3f80-2565-11e9-9456-224563572a84.png)

Outside celery task:
![screen shot 2019-01-31 at 2 31 28 pm](https://user-images.githubusercontent.com/9755598/52083513-3ef9e580-2565-11e9-99d3-102c6de39477.png)

Can you dig into how to set the expiration on the cache while still updating the cache with a celery task?

Reply:
Curious! I'll dig into this. I'm interested in how you found this out, did you just use Data Grip? How can you be sure this is related to Celery?

Reply:
@Johnetordoff, yes, I looked at your osf_cache table in DataGrip and the `expires` date was incorrect.  I'm not _certain_ at all that it's celery, but if I updated the cache like below, instead of in a celery task, the expiration date was correct: 

```python
def update_storage_usage(target):
    Preprint = apps.get_model('osf.preprint')

     if not isinstance(target, Preprint) and not target.is_quickfiles:
         storage_usage_total = target.files.aggregate(sum=models.Sum('versions__size'))['sum'] or 0

         key = cache_settings.STORAGE_USAGE_KEY.format(target_id=target._id)
         cache.set(key, storage_usage_total, cache_settings.FIVE_MIN_TIMEOUT)
```
It's all very strange!

Reply:
@Johnetordoff, I discussed with @sloria, my worker container needed to be restarted - I thought I had done this, but clearly I hadn't.  When pulling down your code again, and restarting the worker container, the correct expiration date was reflected in DataGrip.